src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -440,7 +446,28 @@ export class SearchInterface extends RootComponent implements IComponentBindings
     responsiveSmallBreakpoint: ComponentOptions.buildNumberOption({
       defaultValue: SMALL_SCREEN_WIDTH,
       depend: 'enableAutomaticResponsiveMode'
-    })
+    }),
+    /**
+     * Specifies the search interface responsive mode that should be used.
+     *
+     * When the mode is auto, the width of the window/device that displays the search page is used to determine which layout the search page should use (see [enableAutomaticResponsiveMode]{@link SearchInterface.options.enableAutomaticResponsiveMode}, [responsiveMediumBreakpoint]{@link SearchInterface.options.responsiveMediumBreakpoint} and [responsiveSmallBreakpoint{@link SearchInterface.options.responsiveSmallBreakpoint}])
```

Comment:
That's a pretty long line, could you put the (see [...]) part on a new line?

Reply:
Done 😄